app/mailers/notifier.rb

```diff
@@ -50,8 +50,9 @@ def order_receipt(args)
 
   def review_orders(args)
     @user = User.find(args[:user_id])
-    @accounts = Account.find(args[:account_ids]).map(&:account_list_item).to_sentence
-    send_nucore_mail @user.email, text("views.notifier.review_orders.subject")
+    @accounts = Account.find(args[:account_ids])
+    @facility = args[:facility]
```

Comment:
I'm not 100% of how this is hooked up, but is this method using GlobalID when it gets sent to delayed_job? When delayed_job serializes its arguments, it sometimes will end up serializing the entire facility object, which we like to avoid. I would recommend continuing to pass in the `facility_id`. You could probably do something like `Facility.find_by(id: args[:facility_id])` (I think that will return `nil` if the argument is `nil`) instead of raising an error.